projects/mondo-icd10cm/boomer_input/exact_match_only/prefix.yaml

```diff
@@ -1,8 +1,6 @@
-ICD10CM: http://purl.bioontology.org/ontology/ICD10CM/
-NCI: http://purl.obolibrary.org/obo/NCIT_
+ICD10CM: http://apps.who.int/classifications/icd10/browse/2010/en#/
```

Comment:
Why did this become ICD10CM again? Shouldnt it be whatever is used in icd10cm.owl?

Reply:
That came from the [sssom.tsv file in mondo](https://github.com/monarch-initiative/mondo/blob/master/src/ontology/mappings/mondo_exactmatch_icd10cm.sssom.tsv)

Reply:
I have fixed it in this project for now.